src/components/content/order/migrate/Migrate.tsx

```diff
@@ -26,6 +26,7 @@ export const Migrate = ({ currentSelectedService }: { currentSelectedService: Se
     );
     const [userAvailableServiceVoList, setUserAvailableServiceVoList] = useState<UserAvailableServiceVo[]>([]);
 
+    const [isQueryLoading, setIsQueryLoading] = useState<boolean>(false);
```

Comment:
@Alice1319 Do we really need this extra hook? can we not directly use the `listAvailableServices.isLoading` value?